src/lib.rs

```diff
@@ -809,6 +848,26 @@ pub enum OnlineWalletSubCommand {
         )]
         tx: Option<String>,
     },
+    /// Produce a proof of reserves
+    #[cfg(feature = "reserves")]
+    ProduceProof {
+        /// Sets the message
+        #[structopt(name = "MESSAGE", long = "message")]
+        msg: String,
+    },
+    /// Verify a proof of reserves for our wallet
+    #[cfg(feature = "reserves")]
+    VerifyProof {
+        /// Sets the PSBT to verify
+        #[structopt(name = "BASE64_PSBT", long = "psbt")]
+        psbt: String,
+        /// Sets the message to verify
+        #[structopt(name = "MESSAGE", long = "message")]
+        msg: String,
+        /// Sets the number of block confirmations for UTXOs to be considered. If nothing is specified, 6 is used.
+        #[structopt(name = "CONFIRMATIONS", long = "confirmations")]
+        confirmations: Option<u32>,
```

Comment:
If you specify the `default_value` via struct opt then you don't need an Option here:
 ```suggestion
        confirmations: u32,
```